src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +121,10 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={!newMovie.title
+              || !urlValidator(newMovie.imgUrl)
+              || !urlValidator(newMovie.imdbUrl)
+              || !newMovie.imdbId}
```

Comment:
better to move in constant.

```
const isDisabled = !newMovie.title.trim() || !urlValidator(newMovie.imgUrl) || !urlValidator(newMovie.imdbUrl) || !newMovie.imdbId.trim()
```

```suggestion
            disabled={isDisabled}
```
trim for other required values will solve such problem as on screencast
![chrome-capture-2024-1-20](https://github.com/mate-academy/react_movies-list-add-form/assets/33029864/95444733-5a2b-4aab-b54e-f98a39e92426)